### ISSUE TITLE
can appActivity blank by https://github.com/appium/appium-adb/pull/599

### ISSUE THREAD

**Issue by KazuCocoa** [appium/appium-uiautomator2-driver#528]
just a working memo.
WIth https://github.com/appium/appium-adb/pull/599, maybe we can get rid of appActivity name as a mandatory parameter. If it was blank, we can set the default launchable activity.

**Comment by jlipps** [comment]
yes, absolutely. we should no longer require appActivity to be present since we can now determine launchable activity without it.

**Comment by vrnvorona** [comment]
> No app and appActivity capabilities start a session with do nothing.

I thought from docs that appActivity is required if paired with appPackage (which I need for appium to recognize my id's).

**Comment by KazuCocoa** [comment]
Sorry I missed appPackage. no app, or no appActivity and appPackage, then an appium session starts without starting any activities. Although our general expected use case is with app, or with appActivity and appPackage, and start them in a new session request.
